src/Config/Config.py

```diff
@@ -94,7 +94,7 @@ def set_user(self, user):
         address='http://codeforces.com/api/user.info?handles='+user
         response = requests.get(url=address,proxies=self.get_proxy())
         if(response.status_code!=200):
-            raise UsernameError('User not Found')
+            raise UsernameError('UsernameNotFound')
```

Comment:
I was saying that there is no such class as "UsernameError". You have to create it on your own. Also I think the name of class should be "UsernameNotFound" with error string as "User not found".